.travis.yml

```diff
@@ -15,12 +15,39 @@ addons:
 cache:
   directories:
     - $HOME/.cabal
-ghc:
-  - '8.10'
-  - '8.8'
-  - '8.6'
+    - $HOME/.stack
+    - $TRAVIS_BUILD_DIR/.stack-work
+
+matrix:
+  include:
+
+  # Cabal
+  - ghc: '8.10'
+  - ghc: '8.8'
+  - ghc: '8.6'
+
+  # Stack
+  - ghc: '8.10.4'
+    env: STACK_YAML="$TRAVIS_BUILD_DIR/stack.yaml"
+  - ghc: '8.8.4'
+    env: STACK_YAML="$TRAVIS_BUILD_DIR/stack-lts-16.31.yaml"
+  - ghc: '8.6.5'
+    env: STACK_YAML="$TRAVIS_BUILD_DIR/stack-lts-14.27.yaml"
+
 script:
-  - cabal new-build all --enable-tests
-  - cabal new-test all --enable-tests
-  - git fetch origin master:origin/master && git rebase origin/master --exec "cabal new-build all --enable-tests"
+  - |
+    if [ -z "$STACK_YAML" ]; then
+      cabal new-build all --enable-tests
+      cabal new-test all --enable-tests
+      # git fetch origin master:origin/master && git rebase origin/master --exec "cabal new-build all --enable-tests"
```

Comment:
It would be great if we can re-enable this test

Reply:
There's a conflict when rebasing on master. What's the best way to proceed with dealing with that ? If I rebase my dev_portmidi branch on master and fix the conflict I can't then merge with the current dev_portmidi branch.

Reply:
Probably the conflict is because you've moved the script to a separate file?

Reply:
Also, the PR target should be `master`, not `dev_portmidi`!

Reply:
I meant that when rebasing dev_portmidi on master there are conflicts currently. Got it, I will target master and start a new PR.